docs/source/usage/tutorials/Vanilla Policy Gradient (VPG).md

```diff
@@ -0,0 +1,23 @@
+# Vanilla Policy Gradient (VPG)
```

Comment:
could you rename to file to `using_vpg.md`

In general its better to not include spaces in filesnames

Reply:
Sure